README.md

```diff
@@ -0,0 +1,2 @@
+<h1>survey-page</h1>
+   *Portfolio main page with a mobile and main page*
```

Comment:
The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?